package.json

```diff
@@ -50,7 +50,7 @@
     "docs": "typedoc",
     "jest": "jest --coverageReporters text-summary --",
     "lint": "eslint . --cache --ignore-path .gitignore",
-    "prepare": "npm run build",
+    "prepare": "husky install && npm run build",
```

Comment:
Yea, Husky completely changed how it works after v4, which is one of the reasons I postponed a bit doing the upgrade. They even have a repository just for doing the change: https://github.com/typicode/husky-4-to-7
It's also why the file above is needed. That is now how you provide pre-commit commands. So it's either that or moving to a different library.

Reply:
Still thinking about this since I'm also not a huge fan about the more invasive changes this does (it changes your project git config to point to the husky hook folder explicitly). What do you think about switching to 
 * https://www.npmjs.com/package/pre-commit (not updated in 5 years)
 * https://www.npmjs.com/package/simple-git-hooks (fewer installs)
 * Staying on husky v4 (probably not going to get updates anymore, but maybe not relevant?)

I'm thinking of perhaps not upgrading husky as long as we don't run into any issues.

Reply:
Let's not upgrade indeed; and hope that Husky comes to its senses, or someone creates another package.

Reply:
https://github.com/typicode/husky/discussions/1040

Reply:
and
- https://blog.typicode.com/husky-git-hooks-javascript-config/
- https://blog.typicode.com/husky-git-hooks-javascript-config/

Reply:
Not sure if you're posting this to explain why they did it, or why they probably won't change back, but my one issue with changing the hooksPath in the local git config is that if we ever remove this solution again, the entry remains until you manually remove it.